client/src/common/client.ts

```diff
@@ -297,6 +297,7 @@ export interface DidChangeWatchedFileSignature {
 
 type _WorkspaceMiddleware = {
 	didChangeWatchedFile?: (this: void, event: FileEvent, next: DidChangeWatchedFileSignature) => Promise<void>;
+	applyEdit?: (this: void, params: ApplyWorkspaceEditParams, next: ApplyWorkspaceEditRequest.HandlerSignature) => Promise<ApplyWorkspaceEditResult>;
```

Comment:
For consistency the middleware should be called handleApplyEdit

Reply:
Changed!